ext/bcmath/libbcmath/src/init.c

```diff
@@ -45,8 +45,8 @@ _bc_new_num_ex (length, scale, persistent)
 {
   bc_num temp;
   /* PHP Change:  add length check */
-  if ((size_t)length+(size_t)scale > INT_MAX) {
-   zend_error(E_ERROR, "Result too long, max is %d", INT_MAX);
+  if ((size_t)length + (size_t)scale > INT_MAX) {
+   zend_throw_error(NULL, "Result exceeds the length a PHP string can hold");
```

Comment:
The check doesn't make sense in this form, as execution will continue after you throw. However, you can just drop this check completely, as safe_pemalloc already performs it.